src/core/payouts.ts

```diff
@@ -98,18 +98,19 @@ export async function getPayouts(blocks: Block[], stakers: StakingKey[], totalSt
 
   let payoutJson: PayoutTransaction[] = [];
   let totalPayout = 0;
-  let payoutFileString: string = "(";
   stakers.forEach((staker: StakingKey) => {
-    payoutJson.push({
-      publicKey: staker.publicKey,
-      amount: staker.total,
-      fee: transactionFee
-    });
-    totalPayout += staker.total;
+    const amount = staker.total;
+    if (amount > 0) {
+      payoutJson.push({
+        publicKey: staker.publicKey,
+        amount: amount,
+        fee: transactionFee
```

Comment:
Now that we roll up payouts up at index, we can leave this at 0, and don't need to pass the fee in; we'll want set it outside in the rollup.

Reply:
That will mean we can remove the fee from the function too I think

Reply:
Late comment: maybe rename it to match the env var name more closely so we aren't confused about what it is again - or even more clear e.g. payorSendTransactionFee or something like that.